### PR TITLE
library: Status Page Demo

### PR DESCRIPTION
**Pull request by Datagirlcmr**
Add the files for  a status page
implement a default statuspage

### CODE REVIEW

File: src/Library/demos/Scrollbar/main.blp
Comment (andyholmes) [inline src/Library/demos/Scrollbar/main.blp:1]: This second demo should be moved to a separate branch.

Generally in FOSS communities, and especially GNOME, we follow a "feature branch" workflow. That usually looks something like this:

```sh
# Keep separate remotes with your fork as "origin" and the main repository as "upstream"
git clone git@github.com:Datagirlcmr/Workbench-gnome.git && cd Workbench
git remote add upstream git@github.com:sonnyp/Workbench.git

# Switch the main branch and pull the latest upstream code
git switch main
git pull upstream main

# Switch to a new branch for the feature
git switch -c datagirlcmr/scrollbar-demo
# Push the branch to your remote
git push -u origin datagirlcmr/scrollbar-demo
```

To migrate your current work, you can:

```sh
# Assuming the remotes have been setup as above
git switch main && git pull upstream main
git switch -c datagirlcmr/scrollbar-demo
# This is the actual commit hash from your Scrollbar Demo
git cherry-pick dc70900
```

File: src/Library/demos/Status Page/main.blp
Comment (andyholmes) [inline src/Library/demos/Status Page/main.blp:1]: This file is missing all of the elements referred to in `Status Page/main.js`.

File: src/Library/demos/Status Page/main.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import Gtk from "gi://Gtk?version=4.0";
+
+Gtk.init();
+
+const content = workbench.builder.get_object("content");
+const entry = workbench.builder.get_object("entry_1");
+const button = workbench.builder.get_object("submit_button");
+
+button.connect("clicked", () => {
```
Comment (andyholmes) [inline src/Library/demos/Status Page/main.js:9]: Because this isn't in the BluePrint UI file, this causes an error preventing the demo from loading.

**Review by andyholmes — CHANGES_REQUESTED** [review]
It looks like your demos aren't quite ready to go. 

Please mark your pull requests as "Draft" until they are ready to be reviewed, and be sure to give them a test run in Workbench.

**Comment by Datagirlcmr** [comment]
We can archive this pull request, as I have opened a different one with changes

**Pull request closed without merging**